src/Common/Core/Twig/Extensions/BaseTwigModifiers.php

```diff
@@ -19,15 +19,15 @@ class BaseTwigModifiers
      * Format a number as currency
      *    syntax: {{ $string|formatcurrency($currency, $decimals) }}.
      *
-     * @param string $string   The string to form.
+     * @param float $number The string to form.
      * @param string $currency The currency to will be used to format the number.
-     * @param int    $decimals The number of decimals to show.
+     * @param int $decimals The number of decimals to show.
      *
      * @return string
      */
-    public static function formatCurrency($string, $currency = 'EUR', $decimals = null)
+    public static function formatCurrency(float $number, string $currency = 'EUR', int $decimals = null): string
```

Comment:
I think this will introduce some problems, as there are probably developers who are passing integers as well